glyphs-reader/src/glyphslib_enums.rs

```diff
@@ -0,0 +1,116 @@
+//! Generated by glyphs-reader/data/update.py
+
+/// The primary category for a given glyph
+///
+/// These categories are not the same as the unicode character categories.
+use crate::glyphdata_bundled::BundledEntry;
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+#[repr(u8)]
+pub enum Category {
+    Mark,
+    Space,
+    Separator,
+    Letter,
+    Number,
+    Symbol,
+    Punctuation,
+    Other,
+}
+
+impl BundledEntry for Category {
+    fn element_size() -> usize {
+        1
+    }
```

Comment:
okay so I think we should be able to get by leaving `Category` and `Subcategory` defined by hand, and then just defining this conversion function? As long as we have a unit test that decodes an instance of each discriminant we can have faith in the conversion.

(I have reason to suspect we might need to rework the implementation of these enums in the future to account for arbitrary string values, and that will be significantly more annoying if they are defined in some python codegen file)